.github/CODEOWNERS

```diff
@@ -0,0 +1,14 @@
+# Users referenced in this file will automatically be requested as reviewers for PRs that modify the given paths.
+# See https://help.github.com/articles/about-code-owners/
+
+# ClickOne
+/src/cliconce @NikolaMilosavljevic
+
+# Releases library
+/src/Microsoft.Deployment.DotNet.Releases @joeloff
+
+# Infra
+/eng/DotNetBuild.props @dotnet/source-build @dotnet/product-construction
+/eng/SourceBuild* @dotnet/source-build
+/eng/jobs @NikolaMilosavljevic
```

Comment:
Instead of declaring eng/jobs and azure-pipelines, it would be better to define a default at the top.

`* dotnet/deployment-tools-admins`

This would cover everything that isn't included in one of the explicit paths.

Reply:
Fixed with https://github.com/dotnet/deployment-tools/pull/447/commits/0540e291a325507159b6ed1e57c5e3b12c41fb08